.github/workflows/NUnitConsoleAndEngine.CI.yml

```diff
@@ -31,12 +31,14 @@ jobs:
         with:
           fetch-depth: 0
 
+        # Ensure all required runtimes are available
       - name: 🛠️ Setup .NET
         uses: actions/setup-dotnet@v4
         with:
           dotnet-version: |
             3.1.x
             6.0.x
+            7.0.x
```

Comment:
Why are we adding the 7.0 runtime to remove support for it?

Reply:
A bit of a similar question from me @CharliePoole . I assume these are needed for tests?

Reply:
An earlier build failed because I removed it by mistake, so I added it back. Yes, we need both 3.1 and 7.0 for the test data assemblies, like mock-assembly.dll.